scm/src/CMakeLists.txt

```diff
@@ -105,9 +106,15 @@ if (${CMAKE_BUILD_TYPE} MATCHES "Debug")
     message (FATAL_ERROR "This program has only been compiled with gfortran, pgf90 and ifort. If another compiler is needed, the appropriate flags must be added in ${CMAKE_SOURCE_DIR}/CMakeLists.txt")
   endif (${CMAKE_Fortran_COMPILER_ID} MATCHES "GNU")
 elseif (${CMAKE_BUILD_TYPE} MATCHES "Release")
-  set (CMAKE_C_FLAGS       "-O2 -g -fPIC" CACHE STRING "" FORCE)
-  set (CMAKE_CXX_FLAGS     "-O2 -g -fPIC" CACHE STRING "" FORCE)
-  set (CMAKE_Fortran_FLAGS "-O2 -g -fPIC" CACHE STRING "" FORCE)
+  if(APPLE)
+    set (CMAKE_C_FLAGS       "-fmax-stack-var-size=30000 -O2 -g -I /Volumes/Cluster/opt/local/include/" CACHE STRING "" FORCE)
```

Comment:
I'm curious why these changes were needed. I believe that CMAKE_XXX_FLAGS are propagated to CMakeLists.txt for submodules, include ccpp-physics and ccpp-framework, so I'll probably want to limit using these flags for files that really need them. Also, isn't the -I include directory machine dependent? If this is necessary, we might want to only add this for a specific machine.

Reply:
Any machine-dependent configurations should, where possible, go into scm/etc/MACHINE_setup.{csh,sh} and/or into doc/README_MACHINE.txt (install instructions and some system wide, default settings).

Reply:
Yes, the include file is machine dependent and should be moved out.  
But how do I tell the script to use gfortran compile flags.  CMakeLists.txt has logic for Intel, GNU, PGI comilers under teh Debug build type, but nothing for "Release". I guess instead of 'APPLE' it should be 'GNU'.  
Does this make sense?

Reply:
Yes, you can configure different flags for different compilers in the same way as this is done for the Debug build type. Things like `-fmax-stack-var-size=30000` should go into the machine setup script.